stdlib/experimental/array/array.flux

```diff
@@ -50,4 +50,72 @@ import "array"
 // union(tables: [tags, wildcard_tag])
 // ```
 //
+// deprecated: 0.103.0
 from = array.from
+
+// concat appends two arrays and returns a new array.
+//
+// ## Parameters
+// - arr: First array. This array is not modified.
+// - v: Additional array to append to the first.
```

Comment:
Why did you remove the `This array is not modified.` bits?

Reply:
I didn't think it was necessary, but now that I think about it, it may be. In truth, neither array is modified. The both persist after the operation, but the operation outputs a new array.

Reply:
yeah I wanted to call that out because often other langauges mutate one of the two arrays. I'll add something to that effect.